koku/masu/database/trino_sql/gcp/openshift/reporting_ocpgcpcostlineitem_daily_summary_resource_id.sql

```diff
@@ -38,6 +39,7 @@ CREATE TABLE IF NOT EXISTS {{schema | sqlsafe}}.gcp_openshift_daily_tag_matched_
     instance_type varchar,
     service_id varchar,
     service_alias varchar,
+    data_transfer_direction varchar,
```

Comment:
```suggestion
```
This needs to be added within the creation of `reporting_ocpgcpcostlineitem_project_daily_summary` not `gcp_openshift_daily_tag_matched_temp`
Relevant reference: https://github.com/project-koku/koku/pull/5058/files#diff-d93349900fcf45dec5595ef5c01c5f394889315c807c935062130dcc6b538f8cR138